app/frontend/src/features/profile/edit/ProfileForm.tsx

```diff
@@ -213,101 +196,6 @@ export default function EditProfileForm() {
                 </>
               )}
             />
-            <Controller
-              control={control}
-              defaultValue={user.gender}
-              name="gender"
-              render={({ onChange, value }) => {
-                const other = value === FEMALE || value === MALE ? "" : value;
-                return (
-                  <>
-                    <Typography variant="h2">{GENDER}</Typography>
-                    <RadioGroup
-                      row
-                      aria-label="gender"
-                      name="gender"
-                      value={value}
-                      onChange={(_, value) => onChange(value)}
-                    >
-                      <FormControlLabel
-                        value={FEMALE}
-                        control={<Radio />}
-                        label={FEMALE}
-                      />
-                      <FormControlLabel
-                        value={MALE}
-                        control={<Radio />}
-                        label={MALE}
-                      />
-                      <FormControlLabel
-                        value={other}
-                        control={<Radio />}
-                        checked={value !== FEMALE && value !== MALE}
-                        label={
-                          <TextField
-                            onChange={(event) => onChange(event.target.value)}
-                            defaultValue={other}
-                            value={other}
-                          />
-                        }
-                      />
-                    </RadioGroup>
-                  </>
-                );
-              }}
-            />
-            <Controller
-              control={control}
-              defaultValue={user.pronouns}
-              name="pronouns"
```

Comment:
I think we need to keep this, it's not in the signup. Maybe it should be free-form only?

Reply:
I think she / her and he / him are standard enough for those to be options, and the last one can be free form?